JavaScript/JavaScriptSDK/Context/Session.ts

```diff
@@ -71,7 +71,7 @@ module Microsoft.ApplicationInsights.Context {
                 this.initializeAutomaticSession();
             }
 
-            var now = +new Date;
+            var now = dateTime.Now();
```

Comment:
Actually, we want to use it instead of Date, so it needs to be a number of milliseconds since 1970. 
The correct value is `window.performance.now() + window.performance.timing.navigationStart`.